plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -141,10 +163,13 @@ def main():
         schema=dict(type="str", required=True),
         template=dict(type="str", required=True),
         site=dict(type="str", required=True),
+        l3out_uuid=dict(type="str"),
         l3out=dict(type="str", aliases=["l3out_name"]),
         l3out_schema=dict(type="str"),
         l3out_template=dict(type="str"),
+        l3out_template_id=dict(type="str"),
         l3out_on_apic=dict(type="bool"),
+        l3out_on_l3out_template=dict(type="bool"),
```

Comment:
what happens when l3out_on_apic and l3out_on_l3out_template are both set to true? should we prevent this to be allowed?